bazel/third_party/openssl/openssl_test.cc

```diff
@@ -0,0 +1,68 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Why do we need this file? To make sure things are compiling correctly?

Reply:
Exactly! I'm more than happy to remove it, especially once we have Rust crates setup and openssl is implicitly tested by our Rust tests